docs/src/content/docs/reference/main/stdlib/other.adoc

```diff
@@ -0,0 +1,28 @@
+---
+title: Other Functions
+---
+
+== Length
+
+We offer a generic function to calculate the length/cardinality of elements. 
+
+[source,clojure]
+----
+(length <expr>)
+----
+
+
+[source,sql]
+----
+LENGTH(<expr>)
+----
+
+Where `<expr>` is one of the following:
+
+* A **string** - returns the number of utf8 characters in the string
```

Comment:
As a follow up/follow on work, probably want to line up with however we decide to consistently reference types/type names.